### PR TITLE
Bert baseline for Amazon Reviews

### PR DESCRIPTION
**Pull request by ckarouzos**

### CODE REVIEW

File: slp/modules/classifier.py [hunk 1]
```diff
@@ -14,3 +14,15 @@ def __init__(self, encoder, encoded_features, num_classes):
     def forward(self, *args, **kwargs):
         x = self.encoder(*args, **kwargs)
         return self.clf(x)
+
+class BertClassifier(nn.Module):
+    def __init__(self, encoder, encoded_features, num_classes):
+        super(BertClassifier, self).__init__()
+        self.encoder = encoder
+        self.clf = FF(encoded_features, num_classes,
```
Comment (georgepar) [inline slp/modules/classifier.py]: I think FF might be buggy. Use nn.Linear directly

File: slp/trainer/handlers.py [hunk 2]
```diff
@@ -68,3 +68,15 @@ def attach(self, trainer: Engine, evaluator: Engine,
             Events.EPOCH_COMPLETED,
             self, evaluator, dataloader,
             validation=validation)
+
+class PeriodicNewbob(object):
+    def __init__(self, period):
+        self.period = period
+
+    def __call__(self, engine: Engine, optimizer):
+        if engine.state.epoch % self.period == 0:
```
Comment (georgepar) [inline slp/trainer/handlers.py:81]: There is also the classic Newbob that monitors the val loss and if it stops decreasing it does lr/2

**Review by georgepar — APPROVED** [review]
LGTM
